snippets/emulated-nrf54l10/snippet.yml

```diff
@@ -0,0 +1,12 @@
+#
+# Copyright (c) 2024 Nordic Semiconductor
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+#
+
+name: emulated-nrf54l10
+
+boards:
+  nrf54l15dk/nrf54l15/cpuapp:
```

Comment:
can use a wildcard for this so it works on any board with an nrf54l15 but not required